tests/integration/annotation_import/conftest.py

```diff
@@ -519,13 +623,52 @@ def point_inference(prediction_id_mapping):
 
 
 @pytest.fixture
-def entity_inference(prediction_id_mapping):
+def entity_inference(request, prediction_id_mapping):
```

Comment:
What is `request`?

Reply:
`request` represents the calling function context
I was experimenting with changing fixture data based on whether the caller is export_v1 or _v2... however even if I could get it to work, it would make our code just more complex... so just ended up writing more fixtures
will remove `request`